py-scripts/lf_webpage.py

```diff
@@ -1443,16 +1782,53 @@ def main():
                             test_name=args.test_name,  # FOR WEBGUI
                             get_url_from_file=args.get_url_from_file,
                             file_path=args.file_path,
+                            file_name=args.file_name,
+                            group_name=args.group_name,
+                            profile_name=args.profile_name,
+                            eap_method=args.eap_method,
+                            eap_identity=args.eap_identity,
+                            ieee80211=args.ieee8021x,
```

Comment:
Left similar comment on another commit. Thank you for addressing feedback on the new argument, but in the future, please also modify the corresponding initializer name to eliminate confusion. We can fix later, not going to block on this.